freqtrade/strategy/interface.py

```diff
@@ -332,7 +336,22 @@ def informative_pairs(self) -> ListPairsWithTimeframes:
                             ("BTC/USDT", "15m"),
                             ]
         """
-        return []
+        informative_pairs = []
+        for fn in [getattr(self.__class__, func) for func in dir(self.__class__)
+                   if callable(getattr(self.__class__, func))]:
+            if not getattr(fn, '_is_informative', False):
+                continue
+            timeframe = getattr(fn, '_timeframe', None)
+            asset = getattr(fn, '_asset', None)
+            assert asset is not None
+            assert timeframe is not None
+            if asset:
+                informative_pairs.append(
+                    (f'{asset.upper()}/{self.config["stake_currency"]}', timeframe))
```

Comment:
we'll for sure need to support "non-stake" pairs too.

assume you're trading in BTC markets - you might want to get a "market baseline" by looking at BTC/USDT (or BTC/BUSD) - neither of which will work with this method.